docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -197,55 +200,102 @@ tcld apikey disable --id <api-key-id>
 tcld apikey enable --id <api-key-id>
 ```
 
-## API Keys for Client Connections
+## Managing API Keys for Service Accounts {#serviceaccount-api-keys}
 
-Temporal Cloud supports secure programmatic access through API Key Authentication.
+Global Administrators have the ability to manage API Keys for any Service Account in their Temporal Cloud Account.
+The process to manage API Keys for Service Accounts is very similar to the process of managing API Keys for Users with one difference: Unlike User API Keys, Global Admins are able to generate API Keys for Service Accounts.
+Disabling/enabling, deleting, and viewing API Keys for Service Accounts follows the same process as API Keys for Users.
+
+### Create an API Key for a Service Account
+
+Create an API Key for a Service Account using the Temporal Cloud UI or tcld.
+
+#### Using the Cloud UI
 
-### Overview
+1. Navigate to https://cloud.temporal.io/settings/api-keys
+2. Click the Create API Key button
+3. Click Service Account at the top of the page under "Create an API Key for*"
+4. Select the Service Account you would like to create an API Key in the “Mapped to identityˮ input box.
+   Existing Service Accounts are provided in the “Mapped to Identityˮ box.
+5. Add values for the API Key. This includes:
+
+- name (required)
+- description (optional)
+- Expiration date (required - defaulted to 30 days)
+
+:::note
+
+Temporal currently supports up to a 90-day duration for API Keys.
+
+:::
+
+6. Click the Generate API Key button
+7. You will see the new API keyʼs secret, if the API Key was created successfully.
```

Comment:
```suggestion 
7. You will see the new API Keyʼs secret, if the API Key was created successfully.
```

Reply:
See earlier comments about capitalization.

This is not a step as the user does nothing here. You can make it a step by adding "Confirm". I think, however, the action will be something along the lines of "Confirm that the new API key's secret token has been successfully generated and copy it to a safe place." followed by "The key will not be shown in full form at any other point" or something along those lines (please check for accuracy).